platform-core-module/src/main/java/com/softicar/platform/core/module/email/mailbox/IMailboxConnection.java

```diff
@@ -0,0 +1,20 @@
+package com.softicar.platform.core.module.email.mailbox;
+
+import com.softicar.platform.core.module.server.AGServer;
+import java.util.Collection;
+
+public interface IMailboxConnection extends AutoCloseable {
```

Comment:
I think we should either fully javadoc this type, or not at all.

Reply:
added Javadoc